index.js

```diff
@@ -54,20 +52,4 @@ module.exports = {
   }
 }
 
-function markDbBirth (db) {
-  const STARTED_AT = 'startedAt'
-  var startedAt
-  db.get(STARTED_AT, (err, ts) => {
-    if (err) return console.error(err)
```

Comment:
I think instead we should just remover this line... it's a stupid thing to have in here.

This function is some future-proofing. I've not written methods to use it but in an implementation in Ticktack (which this is informed by) Dominic recorded when "unread" was started being counted so that if you're just isntalling this, ALL OF HISTORY is not marked as unread. This is particularly relevant when you get into unread counters.

Having this would allow us to write methods which let future devs do more....

Reply:
okay but taking that road: `ts` will still be false-y and writing back to `startedAt` outside of the callback.. no-one is reading that either.

Than the whole thing comes down to `db.put(STARTED_AT, Date.Now())`. I'd prefer to merge this in than leaving the whole thing as an uncommented mystery.